code/__DEFINES/DNA.dm

```diff
@@ -56,8 +56,9 @@
 #define DNA_MOTH_MARKINGS_BLOCK 13
 #define DNA_MUSHROOM_CAPS_BLOCK 14
 #define DNA_POD_HAIR_BLOCK 15
+#define DNA_MONKEY_TAIL_BLOCK 16
```

Comment:
I remember we talked about this in the past but I'm still not entirely sold on us having a DNA block for monkey tails (given there's only one monkey tail option that can exist). Is it absolutely necessary for monkey tails to function?

Reply:
It's not needed and I can remove it, I just thought it would be nice to not let them snowflake it, Felinids up until recently also had their own dna block for just the tail and I didn't want monkeys to be excluded without any good reason for it.

Reply:
I just think DNA blocks are a crap system in general tbh

Reply:
:trollface:

Reply:
I actually forgot the full reason but I removed it and now tails just aren't working

![image](https://github.com/tgstation/tgstation/assets/53777086/3fe38dbd-f7d5-444a-8e7a-d3a243bb30fc)

Unfortunately it looks like my past self was smarter than me

Reply:
And this is why I hate DNA blocks